src/utils/scrollToTop.js

```diff
@@ -0,0 +1,13 @@
+const scrollToTop = (event) => {
+  const element = Object.values(event.target)[0].elementType;
+  console.log(element);
+  if (element === 'nav' || element === 'ol' || element === 'li') {
+    window.scrollTo({
+      top: 0,
+      left: 0,
+      behavior: 'smooth',
+    });
+  }
```

Comment:
Not sure we need this conditional now that we're adding the `scrollToTop` to the `nav` rather than the focus boundary - think it should work fine without it?

```suggestion
  window.scrollTo({
    top: 0,
    left: 0,
    behavior: 'smooth',
  });
```

Reply:
Removing the conditional would break the anchor functionality in Learning and Tips pages